### DISCUSSION TITLE
5. Is there anything else you would like to share with us in the open ?

### DISCUSSION THREAD

**Discussion by andreasamsler** [openZH/covid_19#1483] [General]
Be bold, but refrain from judging individuals.

The goal of our initiative has always been to help filling the gaps, and support those who are in charge / addressed, to fix what's necessary.

Together with your immensely important support (which made it necessary from time to time to raise awareness in the open, too), we have come a long way following this constructive approach.

Many thanks for your time, endurance and passion. ❤️

**Comment by balzamas** [comment]
Just thank you very much! Super responsive team, pick up input very quickly.

I'm still thinking about a central ressource for all the measures in the cantons, what would be a smart way. I currently collect them in my database, but maybe a well structured csv would be helpful for others as well? Endpoint is here: https://covidlaws.net/measuremeterdata/chmeasures/ (short description here: https://covidlaws.net/about/)

**Reply by metaodi** [reply]
At least for the Canton of Zurich it already exists: https://opendata.swiss/dataset/covid_19-massnahmen-im-kanton-zuerich
